test/ripper/dummyparser.rb

```diff
@@ -62,6 +63,7 @@ def to_s
   end
 end
 
+module TestRipper
 class DummyParser < Ripper
```

Comment:
Why not prefix with `TestRipper` like `Node` and `NodeList`?

```suggestion
class TestRipper::DummyParser < Ripper
```

Reply:
Because inside methods of this class, there's lots of references to `Node` and `NodeList`, which are now `TestRipper::Node` and `TestRipper::NodeList`. I can either change all the references to make them full constant path or do what I did, which is produces less of a diff so I did it this way. This is also why I didn't change the indentation. But I can do it whichever way you like, not a problem just let me know what you prefer.